### DISCUSSION TITLE
Existing IFC file

### DISCUSSION THREAD

**Discussion by avinash-218** [IfcOpenShell/IfcOpenShell#4827] [General]
Hey Guys,
    Consider I have created an ifc file using ifcopenshell. Can i import the ifc file in blender and can edit on top of it using the blenderbim add on?
    
   For example, I have created a wall using ifcopenshell and exported it as ifc file. Now Can i able to import the file in the blender and add window or wall on it?

**Comment by theoryshaw** [comment]
yes. :)
